README.md

```diff
@@ -94,16 +93,26 @@ Jest tests.
 **Note**: Detailed explanation on how to use the Jest integration can be found
 at [docs/jest-integration.md](docs/jest-integration.md).
 
-A fuzz test in Jest looks similar to the following example:
+A Jest fuzz test, in this case written in TypeScript, looks similar to the
+following example:
 
-```js
-describe("My function", () => {
-	it.fuzz("can be fuzzed", (data) => {
+```typescript
+// file: "Target.fuzz.ts
+// Import the fuzz testing extension to compile TS code.
+import "@jazzer.js/jest-runner/jest-extension";
```

Comment:
I agree, but could not find any way to do so. Do you have an idea?

Reply:
I was hoping you would magically come up with a solution on a second thought :smile: Theoretically, we should import "fuzz" the same way Jest does it with "it" without adding any extra imports to the test. Maybe a @types/jazzer.js or something?

Reply:
A small improvement could be to move the import into the global `globals.d.ts`. Do you prefer that? I updated the example and docs accordingly.

BTW it would be possible to import the definition in the runner itself, but that would still prevent IDEs from picking it up and displaying warnings.